diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/DiktatRule.kt

```diff
@@ -0,0 +1,60 @@
+package org.cqfn.diktat.ruleset.rules
+
+import org.cqfn.diktat.common.config.rules.RulesConfig
+import org.cqfn.diktat.common.config.rules.isRuleEnabled
+import org.cqfn.diktat.ruleset.constants.EmitType
+import org.cqfn.diktat.ruleset.utils.log
+
+import com.pinterest.ktlint.core.Rule
+import org.jetbrains.kotlin.com.intellij.lang.ASTNode
+
+private typealias DiktatConfigRule = org.cqfn.diktat.common.config.rules.Rule
+
+/**
+ * This is a wrapper around Ktlint Rule
+ *
+ * @param id id of the rule
+ * @property configRules all rules from configuration
+ * @property rules warnings that are used in the rule's code
+ */
+@Suppress("TooGenericExceptionCaught")
+abstract class DiktatRule(id: String,
+                          val configRules: List<RulesConfig>,
+                          val rules: List<DiktatConfigRule>) : Rule(id) {
```

Comment:
please rename "rules" to "inspections". Who was writing a white paper? Me or you?

Reply:
Renamed